cmd/der2ascii/writer.go

```diff
@@ -355,6 +355,40 @@ func derToASCIIImpl(out *bytes.Buffer, in []byte, indent int, stopAtEOC bool) []
 					// Emit the remaining as a DER element.
 					derToASCIIImpl(out, elem.body[1:], indent+1, false) // Adds a trailing newline.
 					addLine(out, indent, "}")
+				} else if len(elem.body) >= 1 && len(elem.body) <= 5 && elem.body[0] < 8 {
+					// Convert to a b`` literal when the leading bit is valid and the
+					// number of data octets is at most 4.
+					// 
+					// The 4 octet limit is as above: X.509 incorrectly encodes things
+					// like signatures and RSA moduli as BIT STRING.
```

Comment:
Well, it's also because overly long bit strings aren't particularly helpful on the legibility wrong.